src/app/personside/infotabs/saksoversikt/DokumentKomponent.tsx

```diff
@@ -163,22 +163,22 @@ class DokumentKomponent extends React.Component<Props> {
                 <EtikettGrå>Saksid: {saksid}</EtikettGrå>
             );
 
-        const vedlegg = dokument.vedlegg && dokument.vedlegg.length > 0 ?
+        const dokumentVedlegg = dokument.vedlegg && dokument.vedlegg.length > 0 ?
```

Comment:
Litt tungt å lese denne med `... ?`. Må lete ganske godt for å finne ` : null` lenger ned. Hva med å skrive `dokument.vedlegg && dokument.vedlegg.length > 0 && (<div>...`? Da blir dokumentVedlegg `undefined` istedenfor `null`, men det funker vel greit..?

Reply:
Eventuelt trekke dokumentvedlegg ut i en egen komponent `<DokumentVedleggListe>` el. som brukes rett i render